packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -582,12 +583,14 @@ class DatePicker extends React.Component {
     }
   }
 
-  setPreSelection = (date) => {
+  setPreSelection = (date,type,value) => {
     const isValidDateSelection = date ? isDayInRange(date, this.props.minDate, this.props.maxDate) : true
     if (isValidDateSelection) {
       this.setState({
         preSelection: date
       })
+      type === "month" ? this.updateAriaLiveStatus(getMonthFromDate(date, this.props)) :
+      type === "year" ? this.updateAriaLiveStatus(value, this.props) :
```

Comment:
`updateAriaLiveStatus ` accepts only one arg.

Reply:
This one added 5 years back and followed the same flow, second prop is internally using the date_util's intl prop value without using it is throwing an error as “cannot read property of intl "